src/ducks/AlternativeStore/useAlternativeStore.ts

```diff
@@ -0,0 +1,39 @@
+import { generateI18nConfig } from 'ducks/AlternativeStore/helpers'
+import { transformData } from 'ducks/AlternativeStore/transformData'
+import {
+  AlternativeShortcut,
+  AlternativeStoreConfig
+} from 'ducks/AlternativeStore/types'
+import { buildShortcutsQuery } from 'ducks/queries'
+import { useMemo } from 'react'
+
+import { useQuery } from 'cozy-client'
+import { IOCozyFile } from 'cozy-client/types/types'
+import flag from 'cozy-flags'
+import { useExtendI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+export const useAlternativeStore = (): {
+  alternativeApps: AlternativeShortcut[]
+  installedAlternativeApps: AlternativeShortcut[]
+} => {
+  const shortcutsQuery = buildShortcutsQuery()
```

Comment:
@cballevre do you have an idea how we should handle this scenario?

Reply:
I think I see 2 options : 
- (best one) revert the way to search shortcut by searching folder first by path in the flag and after search for shortcut by dir_id with a `useQueryAll` to ensure having all files
- use the current version and transform the `useQuery` into `useQueryAll` but not optimal at all but it can be a first step